core/iwasm/interpreter/wasm_loader.c

```diff
@@ -1607,6 +1607,12 @@ resolve_func_type(const uint8 **p_buf, const uint8 *buf_end, WASMModule *module,
     wasm_dump_func_type(type);
 #endif
 
+    if (!wasm_func_type_validation(type)) {
```

Comment:
For interpreter mode, the param/result type of func_type has been checked in resolve_value_type (L1517 and L1528), had better not add extra function to check it. Had better enhance resolve_value_type if there is issue found.